src/views/Billing/InvoiceDetailsView.vue

```diff
@@ -104,9 +126,11 @@
   const toast = useToast()
 
   const invoiceData = ref({})
+  const cardDefault = ref({})
 
   onMounted(async () => {
     invoiceData.value = await props.loadInvoiceDataService(route.params.billId)
+    cardDefault.value = await props.loadPaymentMethodDefaultService()
```

Comment:
Evita usar async dentro de lifecycle hook. Pode causar efeitos colaterais inesperados.

Migra esses dois métodos pra um método só e remove o await. Sé for preciso esperar, usa a tag <suspense>